src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.ts

```diff
@@ -46,10 +49,13 @@ export class NavigationComponent implements OnInit, OnDestroy {
     private featureToggles: FeatureTogglesService,
     private telemetryNotificationService: TelemetryNotificationService,
     public prometheusAlertService: PrometheusAlertService,
-    private motdNotificationService: MotdNotificationService
+    private motdNotificationService: MotdNotificationService,
+    public router: Router,
+    private dashboardVersionToggle: DashboardVersionToggleService
   ) {
     this.permissions = this.authStorageService.getPermissions();
     this.enabledFeature$ = this.featureToggles.get();
+    this.dashboardVersion = localStorage.getItem('dashboard_version');
```

Comment:
Instead of locally storing them in browser, i think its best to use the manager module option to store it. That way this won't lose between browsers.

Reply:
Done